instructions.md

```diff
@@ -47,10 +47,12 @@ The plot can be zoomed into by selecting an area to zoom into or by clicking on
 
 ##### Clustering
 
-The clustering method used is based on a distance measure introduced by [Zaliapin and Be-Zion (2013)](https://agupubs.onlinelibrary.wiley.com/doi/pdf/10.1002/jgrb.50179), which defines the distance between two earthquakes in terms of the difference in their location, time of occurrence, and magnitude. The distances are then used to find clusters with fore-, main-, and aftershocks, similarly to an accompanying article [Zaliapin and Be-Zion (2013)](https://agupubs.onlinelibrary.wiley.com/doi/epdf/10.1002/jgrb.50178).
+The clustering method used is based on a distance measure introduced by [Zaliapin and Be-Zion (2013)](https://agupubs.onlinelibrary.wiley.com/doi/pdf/10.1002/jgrb.50179), which defines the distance between two earthquakes in terms of the difference in their location, time of occurrence, and magnitude. The distances are then used to find clusters with fore-, main-, and aftershocks, similarly to an accompanying article [Zaliapin and Be-Zion (2013)](https://agupubs.onlinelibrary.wiley.com/doi/epdf/10.1002/jgrb.50178). The clustering method works by finding the nearest neighbor (using the Zaliapin distance as measure of closeness; the nearest neighbor is also known as the parent event) for each event, this process creates a spanning network; a single cluster where each event is connected to it's parent. the cluster can be modeled as a graph, where each edge is assigned a weight equal to the Zaliapin distance between the two events and the strength of an edge is defined as the inverse of the Zaliapin distance. This graph is pruned by removing weak edges, that is, edges whose strength is lower than certain threshold value, as a result the large graph is split into smaller disjoint ones. 
+
+For each graph, the event with the highest magnitude is labeled as the **mainshock**. Events prior on time to the mainshock are categorized as **foreshocks**, and finally, events taking place after the mainshock, are named **aftershocks**. 
```

Comment:
A couple of small typos:
* 'a single cluster where each event is connected to its parent', instead of it's
* 'Events prior on time to the mainshock', maybe better as 'Events occurring before the mainshock' or something similar
* Capitalization missing from 'the cluster can be modeled as a graph'

Reply:
Got it :ok_hand: